types/plan/plan.go

```diff
@@ -38,7 +38,8 @@ type Spec struct {
 	SourceDir         string `yaml:"sourceDir"`
 	CustomizationsDir string `yaml:"customizationsDir,omitempty"`
 
-	Services map[string][]PlanArtifact `yaml:"services"` //[servicename]
+	Services              map[string][]PlanArtifact `yaml:"services"`                                         //[servicename]
+	TransformersByDefault map[string]string         `yaml:"transformersByDefault,omitempty" m2kpath:"normal"` //[name]filepath
```

Comment:
TransformersByDefault seems non-intuitive. How about something like `InvokedByDefaultTransformers`, `AlwaysRunTransformers` or `TransformersWithNoInput`? Also, since the transformer paths already come in the transformers section, here we can just list the transformer names alone.

Reply:
Thanks for letting me know! 

updated with `InvokedByDefaultTransformers` as it is more relevant as it provides the difference between transformers that are invoked by default or invoked due to services (through directory detect).

modified the section to just show the transformer names